tests/realtikvtest/testkit.go

```diff
@@ -110,8 +111,12 @@ func CreateMockStoreAndDomainAndSetup(t *testing.T, opts ...mockstore.MockTiKVSt
 		tk.MustExec(fmt.Sprintf("set global innodb_lock_wait_timeout = %d", variable.DefInnodbLockWaitTimeout))
 		tk.MustExec("use test")
 		rs := tk.MustQuery("show tables")
+		tables := []string{}
 		for _, row := range rs.Rows() {
-			tk.MustExec(fmt.Sprintf("drop table %s", row[0]))
+			tables = append(tables, fmt.Sprintf("%v", row[0]))
```

Comment:
```suggestion
			tables = append(tables, fmt.Sprintf("`%v`", row[0]))
```

How about adding backticks `` ` `` surrounding `%s` to handle the tables like `xxxx-xxxx`?